src/openMVG/multiview/solver_resection_p5pfr.cpp

```diff
@@ -0,0 +1,373 @@
+/*

```

Comment:
I don't agree. If you let a file license less it means it is not protected.
So I would say, use the right license since the start